frontend/src/css/pages/home.scss

```diff
@@ -0,0 +1,58 @@
+@import '../settings/global';
+@import '../components/_button';
+
+.intro{
+    height: 616px;
+    width: 1280px;
+    margin: auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.importanceOfDiversity{
+    width: 1280px;
+    height: 491px;
+    display: block;
+    margin: auto;
+    justify-content: center;
+    
+}
+.feature{
+    width: 1280px;
+    height: 804px;
+    margin: auto;
+    text-align: center;
+}
+.centreitems{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+
+.list{
```

Comment:
list class should always be used for lists. Do not use it.

Reply:
Okay, got it 👍

Reply:
Removed 👍